### DISCUSSION TITLE
circuit_breaking_exception - No Matter What I try to Access - Standalone Deployment

### DISCUSSION THREAD

**Discussion by rampart51** [Security-Onion-Solutions/securityonion#2838] [General] [answered]
Running v2.3.2 on CentOS 7 with no issues since mid-December 2020.  As of last week, I am seeing circuit_breaking_exception:[parent]Data too large for [<http_request>] would be [3581371698/3.3gb], which is larger than the limit of [3516399616/3.2gb], real usage: [3581369936/3.3gb], new by...  This is occurring with greater frequency on any page that I try to view.

4 Xeon cores 3.00GHz; 16GB memory; 3TB for /nsm with 30% used

Any thoughts on the cause?  I've found that it seems to be Elastic related but am unsure of the fix or the side effects of a fix.

**Comment by weslambert** [comment, marked as answer]
You might consider raising the heap size for Elasticsearch to help with this: https://docs.securityonion.net/en/latest/elasticsearch.html?highlight=heap#heap-size

**Reply by rampart51** [reply]
Appreciate the feedback but /opt/so/saltstack/local/pillar/minions/$minion.sls does not exist.  The only file in minions is XXXXX-XXXXX_standalone.sls.

That file does have an 'esheap' entry.  I assume it is the same but that might be a rather unsafe assumption.
